1-exercises/B-setting-values/exercise2.js

```diff
@@ -24,8 +24,9 @@ let student = {
       exam score is above 60.
   - Use bracket notation to change the value of hasPassed
 */
+if (student["examScore"]>60 && student["attendance"]>=90)
+student["hasPassed"] = true ;
 
```

Comment:
good job, you can also add else :

`else{
   student["hasPassed"] = false;

}`